src/app/lib/config/services/serbian.ts

```diff
@@ -29,7 +29,7 @@ const baseServiceConfig = {
     hasStoryRecommendations: false,
   },
   showAdPlaceholder: false,
-  showRelatedTopics: true,
+  showRelatedTopics: false,
```

Comment:
Don't think we can rely on this unfortunately as this will turn the topics OJ off for CPS assets too, whereas we just want it hidden for this 3 services on Optimo Articles.

Reply:
ahhh okay makes sense, will re think it !

Reply:
Could this be done in the BFF instead of here? It means we could check if the asset is an optimo or CPS one, and if it is optimo, remove the related topics data? Then the article page can't render anything because there is no data.

Alternatively, we do nothing, and just let the related topics be incorrect / 404?

Reply:
Ooo yeah it could be , i wonder if its clearer/simpler keeping it in here though? especially as i think this is a temporary measure until topics transliteration work is completed, it may be simpler to reverse keeping it here. Happy to do it in the BFF though if thats best?

Reply:
I think my preference is:
1. Do nothing
2. hide topics in the BFF
3. hide topics in Simorgh

But happy to get 2nd opinions (because perhaps Harry will need to be notified if we decide to go with #1?)

Reply:
think the decision to hide thes OJs instead of leaving them was made in [this thread](https://bbc-tpg.slack.com/archives/C06ST6MH3QE/p1723115108985379) , but tbf leaving them as is wouldnt break anything , just a confusing user experience.  can spin up a BFF pr to do this work now though !